### PR TITLE
Fixes #18532: Improve documentation on disk space requirement

### PR DESCRIPTION
**Pull request by ncharles**
https://issues.rudder.io/issues/18532

### CODE REVIEW

File: src/reference/modules/installation/pages/requirements.adoc [hunk 1]
```diff
@@ -198,17 +198,17 @@ Please see above for more details about the
...
 Rudder application-related files should stay under 1GB, but the size of the configuration-repository will
 depend of the amount of data you store in it, especially in the shared-files folder (files that will get
-distributed to the agents).
+distributed to the agents). A safe estimation is 1.5GB, plus 10MB per managed nodes.
 
 `/var/log/rudder`::
 Report logs (`/var/log/rudder/reports`) size will depend on the amount of nodes you manage.
 It is possible to reduce this drastically by unticking "Log all reports received to /var/log/rudder/reports/all.log"
 under the *Administration -> Settings* tab in the Rudder web interface. This will prevent Rudder from recording this logs
 in a text file on disk, and will only store them in the SQL database. This saves on space, and doesn't remove any
```
Comment (amousset) [inline src/reference/modules/installation/pages/requirements.adoc:210]: Ce n'est plus vrai en HTTPS, et du coup le volume non plus j'imagine

Comment (ncharles) [reply src/reference/modules/installation/pages/requirements.adoc:210]: most of my disk usage is from non-compliant reports (i have about 95% compliant, so 5% of non compliant, going to non compliant reports).

**Comment by ncharles** [comment]
PR updated with a new commit

**Review by amousset — APPROVED** [review]

**Comment by fanf** [comment]
OK, squash merging this PR

**Pull request merged**